tests/data/test_boyd_study/extract_configs/s3_files_kf-seq-data-hudsonalpha_haib17SB5136_map.py

```diff
@@ -28,6 +28,10 @@
         in_col='file_path',
         out_col=CONCEPT.GENOMIC_FILE.FILE_PATH
     ),
+    keep_map(
+        in_col='file_path',
+        out_col=CONCEPT.GENOMIC_FILE.ID
```

Comment:
We shouldn't have to extract this. The file path is already a unique identifier for a file row in the absence of unique content-based hashes.

Reply:
I don't have anything implemented to allow for multiple unique identifiers (unique keys) per concept. I'll have to add that in the future and then I'll be able to remove this mapping. One step at a time.

Reply:
gotcha